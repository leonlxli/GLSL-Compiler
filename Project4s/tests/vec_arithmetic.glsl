float main() {
	/* tested - and / too */
	float f;
	vec2 v;
	vec2 u;
	vec2 z;
	f = 0.75;
	v.x = u.x = f; // each has 0.75
	v.y = u.y = f * 2.0; // each has 1.5
	z = v + u; // z = (1.5, 3.0)
	return z.x + z.y; // 4.5
}