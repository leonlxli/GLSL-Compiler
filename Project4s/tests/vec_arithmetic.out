Result: 4.500000e+00
